planner/core/errors.go

```diff
@@ -97,5 +97,6 @@ var (
 	ErrAccessDenied        = dbterror.ClassOptimizer.NewStdErr(mysql.ErrAccessDenied, mysql.MySQLErrName[mysql.ErrAccessDeniedNoPassword])
 	ErrBadNull             = dbterror.ClassOptimizer.NewStd(mysql.ErrBadNull)
 	ErrNotSupportedWithSem = dbterror.ClassOptimizer.NewStd(mysql.ErrNotSupportedWithSem)
+	ErrDifferentAsOf       = dbterror.ClassOptimizer.NewStd(mysql.ErrUnknown)
```

Comment:
You can add a new error no in the package `errorno`'s `TiDB self-defined errors.` part. The code may be used for some analysis.

Reply:
We may support it in the future, so we don't need to occupy a place in errorno?